components/Ink/src/private/MDCLegacyInkLayer.m

```diff
@@ -402,11 +399,7 @@ - (void)exit:(BOOL)animated {
     [CATransaction setCompletionBlock:^(void) {
       MDCLegacyInkLayerBackgroundRipple *strongSelf = weakSelf;
       [strongSelf removeFromSuperlayer];
-
-      if ([strongSelf.animationDelegate respondsToSelector:@selector(animationDidStop:
-                                                                           shapeLayer:finished:)]) {
-        [strongSelf.animationDelegate animationDidStop:nil shapeLayer:strongSelf finished:YES];
-      }
+      [strongSelf.animationDelegate animationDidStop:strongSelf];
```

Comment:
Yes it's needed if animated is NO  otherwise we would not get the `animationDidStop:shareLayer:finished:` as far as I see this is the initial intention of the implementation.